### PR TITLE
Fixes DF-1469/Event Detail Venue

### PR DESCRIPTION
**Pull request by sebworks**
Display the Event Detail Venue 

## Changes

- Updates 'meta.less' file to include patterns for displaying the venue.
- Updates '_events.html" file to display the venue and map.
- Updates 'events-posts" file to display the venue and map.
- Updates 'event-post-macros" file to display the venue and map.

## Review

- @jimmynotjim 
- @anselmbradford 

et al

## Preview

event-venue

### CODE REVIEW

File: _includes/event-post-macros.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    {%- set city =  (post.location or 'Washington, DC') -%}    
+    {%- set street = (post.street or '1275 First St NE') -%}
+    {%- set venue = (post.venue or 'Verizon Center') -%}
+    {%- set zip = (post.zip_code or '20002') -%}
+
+    <section class="event-venue">
+        <header class="event-venue_header">
+            <h2 class="event-venue_heading">Washington, DC</h2>
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1-2">
```
Comment (anselmbradford) [inline _includes/event-post-macros.html]: This should include `event-meta` I think? Looks like `event-meta_address` appears below, but it's not enclosed in the rest of the `event-meta` pattern.

File: _includes/event-post-macros.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+   Create an HTML calendar icon when given:
+
+   date:           A valid date.
+
+   additional_classes: Extra classes you wish to add to the icon, space
+                       separated.
+
+   ========================================================================== #}
+
+{% macro calendar_icon(calendar_date, additional_classes) %}
```
Comment (jimmynotjim) [inline _includes/event-post-macros.html]: Are the `additonal_classes` being used?

Comment (sebworks) [inline _includes/event-post-macros.html]: I'm going to add that to the class. I just saw that.

File: _includes/event-post-macros.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+   ==========================================================================
+
+   Description:
+
+   Create an event post summary when given:
+
+   post: A post from a query result.
+
+   path: The path to which post filters are applies.
```
Comment (jimmynotjim) [inline _includes/event-post-macros.html]: Typo, 'applies' should be 'applied'

**Pull request merged**